geth/node/node.go

```diff
@@ -119,8 +119,6 @@ func defaultEmbeddedNodeConfig(config *params.NodeConfig) *node.Config {
 	}
 
 	if config.ClusterConfig.Enabled {
-		// TODO(themue) Should static nodes always be set? Had been done via
```

Comment:
@themue I'm removing this as `PopulateStaticPeers()` is no longer used, do you want me to keep any comment?

Reply:
Nope, is 👍 .